packages/sdks/web-component/src/lib/descope-wc/DescopeWc.ts

```diff
@@ -212,6 +240,7 @@ class DescopeWc extends BaseDescopeWc {
 
     // if there is no execution id we should start a new flow
     if (!executionId) {
+      this.loadSdkScripts();
```

Comment:
Don't you think we should do it once the component is mounted?

Reply:
I don't think so, it has a different lifecycle than the component, it may collect things that is not related to the component, and may actually benefit from loading as soon as possible

Reply:
I think it can be problematic because we might get here more than once 
but we can change it later in case we have issues

Reply:
I think you might be right, but I think that loading it also after mount may create other challenges 

we'll investigate moving further 👍